doc/chapters/service-manifests/index.rst

```diff
@@ -0,0 +1,65 @@
+
+.. _service-manifests:
+
+Service manifests
+*****************
+
+Service manifests are JSON files mapping capabilities to gateway
+configurations. In other words, service manifests describe what
+exact gateway configurations are needed to configure the relevant
+gateways for a specific service in order to realize a capability.
+The service manifests contain the definitions of all the capabilities
+relevant for a service.
+
+For example, a platform might have a capability `com.acme.SomeResource`
+defined. This capability requires access to service `A`. Service `A`
+will then need to deploy a service manifest which contains all gateway
+configurations needed for the `com.acme.SomeResource` capability.
+
+A service manifest might also contain gateway configurations for more
+capabilities.
+
+More services than one might be involved to realize a capability and
+then each relevant service would need to be integrated in this way.
+
+For information about how to work with the manifest files, e.g. where to
+install them etc., see :ref:`Integration <integration>`.
```

Comment:
I feel this phrase is a bit confusing since the next chapter is called "Manifest format". Maybe move this text to the next chapter and add this text as a "disclaimer". E.g. "this chapter describes the format of the Manifest, for information on where to place them and hwo to integrate them, please refer to :ref:`Integration <integration>`.